Samples/WinMLSamplesGallery/WinMLSamplesGallery/SampleMetadata/SampleMetadata.json

```diff
@@ -71,6 +71,17 @@
       "CSharpGithubLink": "https://github.com/microsoft/Windows-Machine-Learning/blob/master/Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/EncryptedModel/EncryptedModel.xaml.cs",
       "Docs": [],
       "IsRecentlyAdded": true
+    },
+    {
+      "Title": "Adapter Selection",
+      "DescriptionShort": "Select different adapters based on your power and performance needs.",
+      "Description": "The sample showcases how to use Windows ML with different adapters that have tradeoffs between power and performance.",
+      "Icon": "\uE155",
+      "Tag": "AdapterSelection",
+      "XAMLGithubLink": "https://github.com/microsoft/Windows-Machine-Learning/blob/user/numform/adapter_selection/Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/AdapterSelection/AdapterSelection.xaml",
```

Comment:
There are various areas in the PR where the link contains a link to your user branch, can we fix it so that it points to master branch?

The files don't exist on master branch yet.. so the links won't work yet

Reply:
Done.